### DISCUSSION TITLE
Ask for what time gpui will be published on crates.io

### DISCUSSION THREAD

**Discussion by JusufGaa** [zed-industries/zed#10431] [Feature Requests]
### Check for existing issues

- [X] Completed

### Describe the feature

I'm writing a library based on gpui, but I can't get it pubilishing on crates.io

```
all dependencies must have a version specified when publishing.
dependency `gpui` does not specify a version
Note: The published dependency will use the version from crates.io,
the `git` specification will be removed from the dependency declaration.
```

**Comment by someone13574** [comment]
Did you try giving it a specific commit hash?

**Comment by git-staus** [comment]
This discussion seems to be a duplicate of the less-specifically worded [Discussion #25033](https://github.com/zed-industries/zed/discussions/25033), which points out the name-squatting of the [`gpui` crate](https://crates.io/crates/gpui) on `crates.io`.

**This is in clear violation of the `crates.io` [usage policy](https://crates.io/policies).**

In addition, the popular [Are we GUI Yet?](https://areweguiyet.com/) website's listing of GPUI also mis-links readers to the empty `crates.io` page. This dead link prompted the [2025 Survey of Rust GUI Libraries](https://www.boringcactus.com/2025/04/13/2025-survey-of-rust-gui-libraries.html#gpui) to bash GPUI quite a bit.

I suggest the maintainers act on this because the current situation is not sustainable!

**Comment by JosephTLyons** [comment]
Thanks for reaching out. We want to better support our community, and the best way for us to do that is through our official Discord. The team is active there and happy to help.

We’re closing the “Help and General Q&A” category on GitHub Discussions and directing all support questions to Discord going forward.

https://zed.dev/community-links
